com.unity.render-pipelines.universal/Editor/ShaderGUI/ShadingModels/LitGUI.cs

```diff
@@ -127,7 +127,9 @@ public static void Inputs(LitProperties properties, MaterialEditor materialEdito
         {
             DoMetallicSpecularArea(properties, materialEditor, material);
             BaseShaderGUI.DrawNormalArea(materialEditor, properties.bumpMapProp, properties.bumpScaleProp);
-            DoHeightmapArea(properties, materialEditor);
+
+            if (HeightmapAvailable(material))
+                DoHeightmapArea(properties, materialEditor);
```

Comment:
Added a check similar to the one for Clear Coat to check if the material has the properties. This was causing a nullreference exceptions when opening up the inspector for Particle Lit shaders.